src/components/AdditionalInfo/AdditionalInfo.stories.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import AdditionalInfo from './AdditionalInfo'
+
+export default {
+  title: 'Compounds/AdditionalInfo',
```

Comment:
@alishaevn should this be changed to Components/AdditionalInfo ?

Reply:
sure should! I wrote this file before I moved them. thanks for the catch!

Reply:
```suggestion
  title: 'Components/AdditionalInfo',
```